xml/System.Diagnostics/ActivityTraceId.xml

```diff
@@ -289,11 +290,12 @@ The characters in <paramref name="idData" /> are not all lower-case hexadecimal
         <Parameter Name="traceId2" Type="System.Diagnostics.ActivityTraceId" />
       </Parameters>
       <Docs>
-        <param name="traceId1">To be added.</param>
-        <param name="traceId2">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="traceId1">The first <see cref="T:System.Diagnostics.Tracing.ActivityTraceId" /> to compare.</param>
+        <param name="traceId2">The second <see cref="T:System.Diagnostics.Tracing.ActivityTraceId" /> to compare.</param>
+        <summary>Determines whether two specified ActivityTraceIds have the same value.</summary>
+        <returns>
+            <see langword="true" /> if the TraceId of <paramref name="traceId1" /> is the different from the TraceId of <paramref name="traceId2" />; otherwise, <see langword="false" />.
```

Comment:
```suggestion
            <see langword="true" /> if the TraceId of <paramref name="traceId1" /> is different from the TraceId of <paramref name="traceId2" />; otherwise, <see langword="false" />.
```